Bottom: ac8a222c909ea86c793e345441db65ef09799351
Top:    8d459a9ed7c382f13b53961f12cc9ff0f78c6ac9
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-02-16 23:51:41 +0000

Adjust STEP export to 2.3mm board thickness (Hack - need something quick!)


---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 1e3a90e..43222ef 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -63,7 +63,7 @@
 #endif
 
 
-#define HACK_BOARD_THICKNESS MM_TO_COORD(1.6)
+#define HACK_BOARD_THICKNESS MM_TO_COORD(2.3)
 #define HACK_MASK_THICKNESS MM_TO_COORD(0.01)
 #define HACK_COPPER_THICKNESS MM_TO_COORD(0.035)
 
diff --git a/src/hid/step/step.c b/src/hid/step/step.c
index 48ddf29..7ac041c 100644
--- a/src/hid/step/step.c
+++ b/src/hid/step/step.c
@@ -45,7 +45,7 @@
 
 #define CRASH fprintf(stderr, "HID error: pcb called unimplemented STEP function %s.\n", __FUNCTION__); abort()
 
-#define HACK_BOARD_THICKNESS MM_TO_COORD(1.6)
+#define HACK_BOARD_THICKNESS MM_TO_COORD(2.3)
 
 
 HID step_hid;
